tests/shell/src/main.c

```diff
@@ -271,6 +272,28 @@ static void dynamic_cmd_get(size_t idx, struct shell_static_entry *entry)
 SHELL_DYNAMIC_CMD_CREATE(m_sub_test_dynamic, dynamic_cmd_get);
 SHELL_CMD_REGISTER(test_dynamic, &m_sub_test_dynamic, NULL, cmd_dynamic);
 
+static void unselect_cmd(void)
+{
+	/* Unselecting command <shell color> */
+	const struct shell *shell = shell_backend_dummy_get_ptr();
+
+	shell->ctx->selected_cmd = NULL;
```

Comment:
If we will keep `ctrl+s` and user will use XOFF/XON mode it will lead to incorrect behaviour:
1. command will be unselected without control of the user
2. connected device will request to stop sending data (ctrl+s esc code) and in response to that we will send a string informing that command has been unselected ;)

Reply:
Regarding adding `alt+r` (`ctrl+s`) in the past. I have considered adding it. Conclusion is that it cannot be done because this test is not calling `shell_process` function that detects incoming charaters.

Reply:
ok